src/core_plugins/metrics/public/directives/visualization.js

```diff
@@ -0,0 +1,42 @@
+import _ from 'lodash';
+import $ from 'jquery';
+import React from 'react';
+import { render, unmountComponentAtNode } from 'react-dom';
+import Visualization from '../components/vis_editor/visualization';
+import addScope from '../lib/add_scope';
+import modules from 'ui/modules';
+import moment from 'moment';
+import createBrushHandler from '../lib/create_brush_handler';
+const app = modules.get('apps/metrics/directives');
+app.directive('metricsVisualization', (timefilter) => {
+  return {
+    restrict: 'E',
+    link: ($scope, $el, $attrs) => {
+      const addToState = ['model', 'visData', 'backgroundColor'];
+      const Component = addScope(Visualization, $scope, addToState);
+      const handleBrush = createBrushHandler($scope, timefilter);
+      render(<Component onBrush={handleBrush} className="dashboard__visualization"/>, $el[0]);
+      $scope.$on('$destroy', () => unmountComponentAtNode($el[0]));
+
+      // For Metrics, Gauges and markdown visualizations we want to hide the
+      // panel title because it just doens't make sense to show it.
+      const panel = $($el[0]).parents('.panel');
+      if (panel.length) {
+        const panelHeading = panel.find('.panel-heading');
+        const panelTitle = panel.find('.panel-title');
+        const matchingTypes = ['metric', 'gauge', 'markdown'];
+        if (panelHeading.length && panelTitle.length && _.contains(matchingTypes, $scope.model.type)) {
```

Comment:
Yes, we want to hide the title for guages, markdown and metrics. The title is redundant for those visualizations and looks very out of place. Keeping the element there but changing the opacity forces the visualizations to be off center. Basically the title is a design flaw with the current dashboard system, it should only be included for visualizations that require it.